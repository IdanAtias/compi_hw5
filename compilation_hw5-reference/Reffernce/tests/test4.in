int semmaTone;
	semmaTone = 8 * 8;
	semmaTone = 8 / 8;
	semmaTone = 1 + semmaTone;
	semmaTone = 2 - semmaTone;
	if ((semmaTone + 3) * 4 == 12)
		print "#8) just affects the ones around me.";
	else
		print "Error #7";