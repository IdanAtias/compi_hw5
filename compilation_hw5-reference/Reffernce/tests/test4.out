#8) just affects the ones around me.
